src/openrct2/paint/Paint.cpp

```diff
@@ -295,36 +295,57 @@ void paint_session_generate(paint_session * session)
     }
 }
 
-static bool is_bbox_intersecting(uint8 rotation, const paint_struct_bound_box& initialBBox,
+template<uint8_t> bool check_bounding_box(const paint_struct_bound_box& initialBBox, 
```

Comment:
`template<uint8_t> static bool check_bounding_box(const paint_struct_bound_box& initialBBox,`

(`static` and remove trailing whitespace)